source/puppet/4.2/reference/lang_exported.markdown

```diff
@@ -34,6 +34,8 @@ Purpose
 
 Exported resources allow nodes to share information with each other. This is useful when one node has information that another node needs in order to manage a resource --- the node with the information can construct and publish the resource, and the node managing the resource can collect it.
```

Comment:
I think this graf wants some revision as well. The key thing I did wrong here was saying they "allow nodes to share information with each other" — it's not _quite_ the nodes doing the sharing, it's the compiler... but it's hard to talk about that. 

Maybe we should say something like,

> Exported resources let you combine information from multiple nodes' catalogs. This is useful when you're managing something that depends on what other nodes are doing. 

?